dotCMS/src/main/java/com/dotmarketing/quartz/job/DistributionJournalCleanup.java

```diff
@@ -17,6 +18,7 @@
     public DistributionJournalCleanup() {
     }
 
+	@CloseDBIfOpened
```

Comment:
Let's change this one with `@WrapInTransaction` and just leave the `APILocator.getDistributedJournalAPI().processJournalEntries();` and remove everything else from this method.